webapp/app/forms/fields.py

```diff
@@ -59,23 +68,35 @@ def pre_validate(self, form):
         ValidElsterCharacterSet().__call__(form, self)
 
 
-class SteuerlotseIntegerField(NumericInputMixin, IntegerField):
+class SteuerlotseIntegerField(NumericInputMaskMixin, IntegerField):
```

Comment:
If you agree with what I wrote above, this should become:
```suggestion
class SteuerlotseIntegerField(NumericInputmodeMixin, NumericInputMaskMixin, IntegerField):
```